packages/SketchMorph2-Core.package/M2MenuItemMorph.class/methodProperties.json

```diff
@@ -10,5 +10,4 @@
 		"isDefaultTool" : "mFr 5/6/2021 15:56",
 		"isDefaultTool:" : "mFr 5/6/2021 15:53",
 		"isM2MenuItemMorph" : "mFr 5/19/2021 09:04",
-		"isSelected:" : "ms 5/13/2021 19:21",
```

Comment:
Any reason why this was removed? (Same as in #61)

Reply:
Since the method no longer exists, I assume this line survived because of our rebases. Squeak then saw that it has a line without a reference in it, and removed it on the next save.

Reply:
Makes sense.